### PR TITLE
Correct flannel cert variables.

### LINKED ISSUE #2322: Installation fails when we select flannel network to deploy with etcd host
Installation is failing with following error:

```
ok: [jupiter-vm1144.pok.stglabs.ibm.com -> jupiter-vm1108.pok.stglabs.ibm.com] => {"changed": false, "invocation": {"module_args": {"checksum_algorithm": "sha1", "follow": false, "get_checksum": true, "get_md5": true, "mime": false, "path": "/etc/origin/node/ca/ca.crt"}, "module_name": "stat"}, "stat": {"exists": false}}

TASK [etcd_client_certificates : fail] *****************************************
task path: /root/openshift-ansible/openshift-ansible/roles/etcd_client_certificates/tasks/main.yml:9
fatal: [jupiter-vm1144.pok.stglabs.ibm.com]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_args": {"msg": "CA certificate /etc/origin/node/ca/ca.crt doesn't exist on CA host jupiter-vm1108.pok.stglabs.ibm.com. Apply 'etcd_ca' role to jupiter-vm1108.pok.stglabs.ibm.com.\n"}, "module_name": "fail"}, "msg": "CA certificate /etc/origin/node/ca/ca.crt doesn't exist on CA host jupiter-vm1108.pok.stglabs.ibm.com. Apply 'etcd_ca' role to jupiter-vm1108.pok.stglabs.ibm.com.\n"}

NO MORE HOSTS LEFT *************************************************************
	to retry, use: --limit @playbooks/byo/config.retry

PLAY RECAP *********************************************************************
jupiter-vm1108.pok.stglabs.ibm.com : ok=104  changed=36   unreachable=0    failed=0   
jupiter-vm1144.pok.stglabs.ibm.com : ok=352  changed=89   unreachable=0    failed=1   
jupiter-vm931.pok.stglabs.ibm.com : ok=152  changed=42   unreachable=0    failed=0   
localhost                  : ok=13   changed=8    unreachable=0    failed=0   

[root@jupiter-vm1108 openshift-ansible]#
```

The content of /etc/ansible/hosts file

```
# Create an OSEv3 group that contains the masters and nodes groups
[OSEv3:children]
masters
nodes
etcd

# Set variables common for all OSEv3 hosts
[OSEv3:vars]
# SSH user, this user should allow ssh based auth without requiring a password
ansible_ssh_user=root

# If ansible_ssh_user is not root, ansible_sudo must be set to true
#ansible_sudo=true

deployment_type=origin
openshift_use_openshift_sdn=False
openshift_use_flannel=True

# uncomment the following to enable htpasswd authentication; defaults to DenyAllPasswordIdentityProvider
#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
osn_storage_plugin_deps=[]
#oreg_url=jupiter-vm1323.pok.stglabs.ibm.com/openshift/origin-${component}:${version}

[etcd]
jupiter-vm1108.pok.stglabs.ibm.com

# host group for masters
[masters]
jupiter-vm1144.pok.stglabs.ibm.com

# host group for nodes, includes region info
[nodes]
jupiter-vm1144.pok.stglabs.ibm.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}" openshift_scheduleable=True
jupiter-vm931.pok.stglabs.ibm.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
```

and the content of /etc/etcd folder in etcd host is

```
[root@jupiter-vm1108 etcd]# tree
.
├── ca
│   ├── ca.crt
│   ├── ca.key
│   ├── certs
│   │   ├── 01.pem
│   │   ├── 02.pem
│   │   └── 03.pem
│   ├── crl
│   ├── fragments
│   │   ├── openssl_append.cnf
│   │   └── openssl.cnf
│   ├── index.txt
│   ├── index.txt.attr
│   ├── index.txt.attr.old
│   ├── index.txt.old
│   ├── openssl.cnf
│   ├── serial
│   └── serial.old
├── ca.crt
├── etcd.conf
├── etcd.conf.2016-08-18@12:56:59~
├── generated_certs
│   ├── etcd-jupiter-vm1108.pok.stglabs.ibm.com
│   │   ├── ca.crt
│   │   ├── peer.crt
│   │   ├── peer.csr
│   │   ├── peer.key
│   │   ├── server.crt
│   │   ├── server.csr
│   │   └── server.key
│   ├── etcd-jupiter-vm1108.pok.stglabs.ibm.com.tgz
│   ├── openshift-master-jupiter-vm1144.pok.stglabs.ibm.com
│   │   ├── master.etcd-ca.crt
│   │   ├── master.etcd-client.crt
│   │   ├── master.etcd-client.csr
│   │   └── master.etcd-client.key
│   └── openshift-master-jupiter-vm1144.pok.stglabs.ibm.com.tgz
├── peer.crt
├── peer.csr
├── peer.key
├── server.crt
├── server.csr
└── server.key

7 directories, 36 files
[root@jupiter-vm1108 etcd]# 
```

### PR DESCRIPTION
**Pull request by abutcher**
Fixes #2322

### CODE REVIEW

**Comment by abutcher** [comment]
@sdodson PTAL

File: roles/flannel/README.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ to 0.3.
...
-| etcd_conf_dir       | {{ openshift.common.config_base }}/node | SSL certificates directory                    |
-| etcd_peer_ca_file   | {{ etcd_conf_dir }}/ca.crt              | SSL CA to use for etcd                        |
-| etcd_peer_cert_file | Openshift SSL cert                      | SSL cert to use for etcd                      |
-| etcd_peer_key_file  | Openshift SSL key                       | SSL key to use for etcd                       |
+| Name                 | Default value                           | Description                                   |
+|----------------------|-----------------------------------------|-----------------------------------------------|
+| flannel_interface    | ansible_default_ipv4.interface          | interface to use for inter-host communication |
+| flannel_etcd_key     | /openshift.com/network                  | etcd prefix                                   |
+| etcd_hosts           | etcd_urls                               | a list of etcd endpoints                      |
+| etcd_cert_config_dir | {{ openshift.common.config_base }}/node | SSL certificates directory                    |
```
Comment (sdodson) [inline roles/flannel/README.md]: Can't find anywhere that this is actually used?

Comment (abutcher) [inline roles/flannel/README.md]:
```
playbooks/common/openshift-node/config.yml:    etcd_cert_config_dir: "{{ openshift.common.config_base }}/node"
```

Comment (abutcher) [inline roles/flannel/README.md]: It should probably be set in defaults for this role but it is set for override ^ there.

Comment (sdodson) [inline roles/flannel/README.md]: It's used only by the etcd_client_certificates role dependency?

Comment (abutcher) [inline roles/flannel/README.md]: It's used by the `etcd_client_certificates` role / family of roles (dep of this role).

**Comment by sdodson** [comment]
LGTM

**Pull request merged**